### ISSUE TITLE
Hanging on startup

### ISSUE THREAD

**Issue by SlimeDog** [lucko/LuckPerms#843]
Spigot 1.12.2
LP 4.1.40

Only change to system was updating LP. Nothing in console/log after the following:

```
[11:08:53 INFO]: [LuckPerms] Enabling LuckPerms v4.1.40
[11:08:53 INFO]:                __        __   ___  __         __
[11:08:53 INFO]:     |    |  | /  ` |__/ |__) |__  |__)  |\/| /__`
[11:08:53 INFO]:     |___ \__/ \__, |  \ |    |___ |  \  |  | .__/
[11:08:53 INFO]:
[11:08:53 INFO]:   Loading version v4.1.40 on Bukkit - CraftBukkit
[11:08:53 INFO]:   Running on server version git-Spigot-c3093ef-7e94e65 (MC: 1.12.2) - 1.12.2-R0.1-SNAPSHOT
[11:08:53 INFO]:
[11:08:53 INFO]: [LP] Loading configuration...
[11:08:53 INFO]: [LP] Found lang.yml - loading messages...
[11:08:53 INFO]: [LP] Loading storage provider... [YAML]
```

Waited 5 minutes. Killed everything off and reverted to LP 4.1.37. All OK.

**Comment by lucko** [comment]
Yeah, seems to be an issue after the latest set of changes.

I can't work out why the hell it's happening though. Seems to be an issue with log4j.

https://paste.ubuntu.com/p/p78wWGP9dV/

**Comment by lucko** [comment]
Okay, fixed in 0f99a6c5caef480e6db58240c89a5296a8dc8a87

**Comment by SlimeDog** [comment]
That was a pretty fast turnaround for "can't work out ..."
